crypto/internal.h

```diff
@@ -501,6 +501,23 @@ static inline uint32_t from_be_u32_ptr(const uint8_t *data) {
 #endif
 }
 
+
+/* to_be_u32_ptr transforms 32-bit value |x| to big-endian-encoded and sets it
+ * at |data| */
+static inline void to_be_u32_ptr(uint8_t *data, uint32_t x) {
```

Comment:
Alternatively, I could have this just return a big-endian-encoded `uint32_t` and each caller is responsible for inserting it into `data`